components/webpack/path-map.js

```diff
@@ -38,5 +38,8 @@ module.exports = function (genPath) {
     // TODO(petemill): Rename 'brave/components/common' dir to
     // 'brave/components/web-common'
     '$web-common': path.resolve(__dirname, '../common'),
+    // react-markdown uses this deep in its tree, and the browser variant uses innerHTML, conflicting with UntrustedWebUI instances
```

Comment:
It's not only UntrustedWebUI that causes this issue - it's the reliance on Trusted Types which disallows innerHTML, right?

Reply:
It's the CSP, yup. This implementation should be faster if bundled with the browser anyway.
In that library they were using `innerHTML` to not bundle the lookup table and not pay the download cost, each time you download that library.

If we disable `trusted-types` we might get other exploitation vectors too.

Reply:
fixed comment